go-backend/db/core_value.go

```diff
@@ -0,0 +1,144 @@
+package db
```

Comment:
where are the test cases of DB methods ? these should be integration test case, which makes actual call to DB.